src/governance/governance.h

```diff
@@ -95,17 +95,14 @@ class CRateCheckBuffer
 
     int GetCount() const
     {
-        int nCount;
         if (fBufferEmpty) {
             return 0;
         }
         if (nDataEnd > nDataStart) {
-            nCount = nDataEnd - nDataStart;
+            return nDataEnd - nDataStart;
         } else {
-            nCount = RATE_BUFFER_SIZE - nDataStart + nDataEnd;
+            return RATE_BUFFER_SIZE - nDataStart + nDataEnd;
```

Comment:
I'd prefer to have one of these `return`s as a default one. Or maybe add `assert(false);` at the end of the method instead.

Reply:
why? It's an if else, one of them is going to happen :D

Reply:
Yeah, so `else` isn't really needed, right? Could be just
```
if (A) {
    return B;
}
return C;
```
 right?

Having a default return path just feels safer imo :)

Reply:
https://github.com/dashpay/dash/pull/3958/commits/a27a5a5945711cebdb53d8931b006800bb0ccf60